au3/libraries/au3-cloud-audiocom/sync/CloudProjectsDatabase.cpp

```
@@ -113,6 +117,13 @@ CloudProjectsDatabase& CloudProjectsDatabase::Get()
     return instance;
 }
 
+bool CloudProjectsDatabase::DatabaseExists()
```

Cppcheck (2.20.0)

[style] The function 'DatabaseExists' is never used.

(unusedFunction)